templates/taxi/driver_list.html

```diff
@@ -20,6 +22,12 @@
         <td>{{ driver.first_name }}</td>
         <td>{{ driver.last_name }}</td>
         <td>{{ driver.license_number }}</td>
+        <td>
+          <a style="color: red"
+            href="{% url 'taxi:driver-delete' pk=driver.id %}">
+            Delete
+          </a>
```

Comment:
Consider adding a confirmation step for the delete action to prevent accidental deletions. This can be done using JavaScript to prompt the user for confirmation.